os/net/lwip/src/core/pbuf.c

```diff
@@ -286,18 +286,19 @@ struct pbuf *pbuf_alloc(pbuf_layer layer, u16_t length, pbuf_type type)
 
 		/* If pbuf is to be allocated in RAM, allocate memory for it. */
 		p = (struct pbuf *)mem_malloc(alloc_len);
-	}
 
-	if (p == NULL) {
-		return NULL;
-	}
+		if (p == NULL) {
```

Comment:
Could you make separation the commit between TCs and network codes?

Reply:
this is not any network code. this is just indentation correction.
so i think it should not be problem.

Reply:
That's the problem on commit side, not on operation side. The changes in network folder is not for tc implementation. You should not include two different kinds of change in a commit. As Jeongchan said, please split them as a different commit.

Reply:
done